src/test/java/cat/udl/eps/entsoftarch/textannot/steps/MetadataTemplateMetadataFieldRelacion.java

```diff
@@ -0,0 +1,48 @@
+package cat.udl.eps.entsoftarch.textannot.steps;
+
+import cat.udl.eps.entsoftarch.textannot.domain.MetadataField;
+import cat.udl.eps.entsoftarch.textannot.domain.MetadataTemplate;
+import cat.udl.eps.entsoftarch.textannot.repository.MetadataFieldRepository;
+import cat.udl.eps.entsoftarch.textannot.repository.MetadataTemplateRepository;
+import cucumber.api.PendingException;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.junit.Assert;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+public class MetadataTemplateMetadataFieldRelacion {
+    @Autowired
+    private MetadataTemplateRepository mtr;
+
+    @Autowired
+    private MetadataFieldRepository mtfr;
+
+    private List<MetadataTemplate> MetadataTemplatesList;
+
+    @Given("^A MetadataTemplate with name \"([^\"]*)\" defines a MetadataField with name \"([^\"]*)\" and type \"([^\"]*)\"$")
+    public void aMetadataTemplateWithNameDefinesAMetadataFieldWithNameAndType(String name, String FName, String Ftype) throws Throwable {
+        MetadataTemplate mt = new MetadataTemplate();
+        mt.setName(name);
+        MetadataField mf = new MetadataField(FName, Ftype);
+
+        mt = mtr.save(mt);
+        mf.setDefinedIn(mt);
+        mtfr.save(mf);
+    }
+
+    @When("^I find MetadataTemplate by MetadataField name \"([^\"]*)\" and type \"([^\"]*)\"$")
+    public void iFindMetadataTemplateByMetadataFieldNameAndType(String FName, String Ftype) throws Throwable {
+        this.MetadataTemplatesList = mtr.findByDefinesNameAndDefinesType(FName, Ftype);
```

Comment:
This feature will be accepted until the current commit. However, for the second sprint, consider performing a call through the API to the findByDefinesNameAndDefinesType method, using the MockMVC and a GET request instead of directly calling the method in the repository.